data/json/npcs/beggars/BEGGAR_3_Luo_Meizhen.json

```diff
@@ -0,0 +1,241 @@
+[
+  {
+    "type": "effect_type",
+    "id": "insulted_luo",
+    "name": [ "Insulted" ],
+    "desc": [ "Oh, you went there." ]
+  },
+  {
+    "type": "npc",
+    "id": "refugee_beggar3",
+    "//": "Angry beggar in the refugee center.",
+    "name_unique": "Luo Meizhen",
+    "gender": "female",
+    "name_suffix": "beggar",
+    "class": "NC_BEGGAR_3",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_REFUGEE_BEGGAR_3",
+    "faction": "wasteland_scavengers"
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_BEGGAR_3",
+    "name": "Beggar",
+    "job_description": "I'm just trying to survive.",
+    "common": false,
+    "//": "All the beggars have been camping inside the evac shelter because they're too weak and unskilled to make it on their own.",
+    "bonus_str": { "rng": [ -4, -1 ] },
+    "bonus_dex": { "rng": [ -4, -1 ] },
+    "bonus_int": { "rng": [ 2, 5 ] },
+    "bonus_per": { "rng": [ 0, 3 ] },
+    "worn_override": "NC_BEGGAR_3_worn",
+    "carry_override": "EMPTY_GROUP",
+    "weapon_override": "EMPTY_GROUP",
+    "traits": [
+      { "trait": "SLOWHEALER2" },
+      { "trait": "SLOWRUNNER" },
+      { "trait": "HEAVYSLEEPER" },
+      { "trait": "FLIMSY2" },
+      { "trait": "FASTLEARNER" },
+      { "trait": "PACIFIST" },
+      { "trait": "Exp_Mycology2" },
+      { "group": "Appearance_demographics" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_BEGGAR_3_worn",
+    "subtype": "collection",
+    "items": [
+      { "item": "hoodie" },
+      { "item": "jacket_evac" },
+      { "item": "jeans" },
+      { "item": "panties" },
+      { "item": "camisole" },
+      { "item": "bra" },
+      { "item": "socks" },
+      { "item": "lowtops" },
+      { "item": "gloves_light" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_REFUGEE_BEGGAR_3",
+    "dynamic_line": {
+      "npc_has_effect": "insulted_luo",
+      "yes": "Fuck off, dickwaddle.",
+      "no": {
+        "u_has_var": "luo_recruited",
+        "yes": "Yo.  Anyone else keen on moving from this bus stop to your tent city?",
+        "no": {
+          "npc_has_var": "dont_insult_luo",
+          "yes": {
+            "npc_has_var": "apology_luo",
+            "yes": {
+              "npc_has_effect": "beggar_has_eaten",
+              "yes": "Hey there.  Good to see you again.",
+              "no": "Careful, I'm getting hangry again and am not totally responsible for my own actions."
```

Comment:
Hungry or angry?

Reply:
"hangry" is a slang word that means angry because you're hungry